src/auth/auth.controller.ts

```diff
@@ -54,4 +61,90 @@ export class AuthController {
 
     res.sendStatus(HttpStatus.OK);
   }
+
+  @ApiExcludeEndpoint()
+  @Get('login')
+  async jwtLogin(
+    @Query(
+      new ValidationPipe({
+        errorHttpStatusCode: HttpStatus.UNPROCESSABLE_ENTITY,
+        transform: true,
+      }),
+    )
+    { token }: LoginQueryDto,
+    @Res() res: Response,
+  ): Promise<void> {
+    const redirectUrl = `${this.config.get<string>(
+      'INCENTIVIZED_TESTNET_URL',
+    )}/leaderboard`;
+    if (this.config.get<boolean>('DISABLE_LOGIN')) {
+      res.redirect(`${redirectUrl}?toast=${btoa('Login is disabled')}`);
```

Comment:
Can you add &persist=true so the error message will stay? Current default behavior is disappear in like 2 second, thats too short for user to take screenshot and share us the error message if they have trouble logging in.